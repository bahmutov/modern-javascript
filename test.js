/* eslint-env jest */
describe('add', () => {
  const add = require('.').add
  it('adds numbers', () => {
    expect(add(1, 20)).toMatchSnapshot()
    expect(add(100, -50)).toMatchSnapshot()
  })
})
describe('sub', () => {
  const sub = require('.').sub
  it('subs numbers', () => {
    expect(sub(1, 2)).toBe(-1)
  })
})
