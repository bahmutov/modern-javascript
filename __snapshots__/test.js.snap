// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add adds numbers 1`] = `3`;

exports[`add adds numbers 2`] = `50`;
